src/ImageUtils.js

```diff
@@ -95,8 +95,8 @@
 
         function doBicubicScale(srcImg, scale) {
 
-            var destWidth = srcImg.width * scale;
-            var destHeight = srcImg.height * scale;
+            var destWidth = Math.round(srcImg.width * scale);
```

Comment:
It is don't make any sense, because doBicubicScale doesn't exist anymore and I use doBicubicInterpolation instead.